plugins/catalog-backend/src/stitching/Stitcher.test.ts

```diff
@@ -206,6 +223,15 @@ describe('Stitcher', () => {
 
       expect(entities[0].hash).not.toEqual(firstHash);
       expect(entities[0].hash).toEqual(entity.metadata.etag);
+      expect(entity.metadata.etag).toEqual(entities[0].hash);
+      const last_updated_at_after_restitch = entities[0].last_updated_at;
+      expect(last_updated_at_after_restitch).not.toBeNull();
+      const msAfterRestitch = DateTime.fromMillis(
+        last_updated_at_after_restitch ? +last_updated_at_after_restitch : 0,
+      )
+        .diff(timeBeforeRestitch, 'milliseconds')
+        .toObject();
```

Comment:
use `.as('milliseconds')` on Duration

Reply:
thanks for advice. It's changed to expect the value of last_updated_at to be not null.